df.md

```diff
@@ -0,0 +1,49 @@
+# 🦈 Senior Product Developer - Data Farming 
+[Ocean Protocol](https://www.oceanprotocol.com) builds tools for people to reclaim control of their data and unleash an open data economy. Ocean tools make it easy to publish & consume data services as data access tokens, and bridge data to DeFi. 
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
+
+
+## Responsibilities
+* Candidate will have to communicate with different teams to sync on roadmap delivery
+* Candidate will need to engage with the community to get faster feedback to understand usage patterns, pain points, and user experience
+* They will need to understand to differentiate good and bad feedback and find real pain points and needed improvements
+* They will improve and evolve product/program based on learnings from usage data and user feedback 
+* They will take ownership of related tech like scripts and manage and update scripts and reward functions to implement design changes and effectively communicate to the community about changes
+* They will have to execute scripts to automate some web3 tasks like token distribution
+* They will have to identify fraudulent actors taking unfair advantage of the program and make improvements to curb them
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
+
+
+## Skills and Experience 
+* We don’t expect applicants to be an expert in everything listed below which is why we’ve separated the requirements into two tiers. The ideal candidate loves to communicate, to code, to build meaningful technical communities and relationships with key stakeholders.
+
+
+## Minimum Qualifications
+* 3+ years of experience working in technical/development roles in the crypto projects
+* 2+ years Hands-on experience in development in Web3 and Web2 tech stack
+* Proven experience in building meaningful technical communities around projects
+* Ability to build and manage relationships with key stakeholders
+* Ability to take calculated decisions based on data 
+* Process-driven, proactive - able to report on your activity and communicate well with the team.
+* Comfortable working autonomously and within a team - our team is small, so balancing independence and collaboration is important.
+* Creative, with extensive writing/public speaking experience
```

Comment:
Everyone will say they're "creative". We don't need this hear.

And we don't need extensive public speaking experience for this role.

I recommend to (a) remove this line, and (b) in "Nice to haves" add "You’re an excellent writer."